core/src/main/java/org/mule/construct/AbstractPipeline.java

```diff
@@ -356,7 +355,7 @@ protected void doStart() throws MuleException
 
     private void createFlowMap()
     {
-        if (!flowMap.isEmpty())
+        if (flowMap != null)
         {
             logger.warn("flow map already populated");
```

Comment:
I wouldn't agree that this is a warning from the user perspective.  Not sure a log statement is even needed here